editor/scene_tree_dock.cpp

```diff
@@ -1620,6 +1621,56 @@ void SceneTreeDock::_selection_changed() {
 	_update_script_button();
 }
 
+void SceneTreeDock::_do_create(Node* p_parent) {
+	Object *c = create_dialog->instance_selected();
+
+	ERR_FAIL_COND(!c);
+	Node *child = Object::cast_to<Node>(c);
+	ERR_FAIL_COND(!child);
+
+	editor_data->get_undo_redo().create_action(TTR("Create Node"));
+
+	if (edited_scene) {
+
+		editor_data->get_undo_redo().add_do_method(p_parent, "add_child", child);
+		editor_data->get_undo_redo().add_do_method(child, "set_owner", edited_scene);
+		editor_data->get_undo_redo().add_do_method(editor_selection, "clear");
+		editor_data->get_undo_redo().add_do_method(editor_selection, "add_node", child);
+		editor_data->get_undo_redo().add_do_reference(child);
+		editor_data->get_undo_redo().add_undo_method(p_parent, "remove_child", child);
+
+		String new_name = p_parent->validate_child_name(child);
+		ScriptEditorDebugger *sed = ScriptEditor::get_singleton()->get_debugger();
+		editor_data->get_undo_redo().add_do_method(sed, "live_debug_create_node", edited_scene->get_path_to(p_parent), child->get_class(), new_name);
+		editor_data->get_undo_redo().add_undo_method(sed, "live_debug_remove_node", NodePath(String(edited_scene->get_path_to(p_parent)) + "/" + new_name));
+
+	}
+	else {
```

Comment:
`} else {`, without the newline in between `}` and the `else {` as per the code style guidelines.

https://travis-ci.org/godotengine/godot/jobs/425751969#L533
http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html

Reply:
Fixed